### DISCUSSION TITLE
Let's Encrypt certificate expiration notice

### DISCUSSION THREAD

**Discussion by SteveTheMonkey12** [cat-box/aws-foundry-ssl#39] [Q&A]
Hey,

I've been using foundry on aws for a year without a hitch and I get an email through from Let's Encrypt Expiry Bot <expiry@letsencrypt.org> saying my security certificate is going to expire in 19 days for their 90 day certificate, which seems odd to me as I've never had one before. Just wondering how to resolve and if it's phishing or not.

Thanks in advance for any help.

**Comment by cat-box** [comment]
It's not spam. Let's Encrypt is the service used to provide the website with the security certificates. The deployment should auto-renew 30 days from expiry. 

There was a bug preventing the auto-renewal which was addressed in this [patch](https://github.com/cat-box/aws-foundry-ssl/wiki/Patches#certificate-renewal). It needs to be manually applied but **only** if you deployed prior to **Jan 15th, 2021**. 

Otherwise, I'm not sure why it's not auto-renewing, you may need to manually renew it through SSH. Send me a DM/message on reddit and I can send you commands to run in your terminal as well as take a look at your configuration.
